### ISSUE TITLE
Window.Visible property throws when window is closed

### ISSUE THREAD

**Issue by stevenbrix** [microsoft/microsoft-ui-xaml#9524]
### Describe the bug

it's strange that this API throws an exception when the window has been closed, i would expect it to just return false. and since there is no way to query that the window is in a state where it's closed, there is no way to check that the window is in an OK state to call it other than keeping separate app state around

### Steps to reproduce the bug

1. close a window
2. after window has closed, check the `Visible` property

### NuGet package version

WinUI 3 - Windows App SDK 1.5.1: 1.5.240311000

**Comment by HO-COOH** [comment]
> since there is no way to query that the window is in a state where it's closed

**Comment by stevenbrix** [comment]
>Isn't there a Window.Closed [event](https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.window.closed?view=windows-app-sdk-1.5#microsoft-ui-xaml-window-closed) which you can handle?

Yes, but the `Window` isn't a `DependencyObject` so you can't add extra properties to it. you have to do all your own bookkeeping to manage window state of the app, which seems silly
